src/workerd/server/server.c++

```diff
@@ -1892,6 +1893,17 @@ static kj::Maybe<WorkerdApiIsolate::Global> createBinding(
       });
     }
 
+    case config::Worker::Binding::FROM_ENVIRONMENT: {
+      const char* value = getenv(binding.getFromEnvironment().cStr());
+      if (value == nullptr) {
+        // TODO(cleanup): Maybe make a Global::Null? (Can't use nullptr_t in OneOf.) For now,
+        // using JSON gets the job done hackily.
+        return makeGlobal(Global::Json { kj::str("null") });
```

Comment:
Setting the value to an empty string would at least have the value test falsy.

```
console.log('' || 'hello') // 'hello'
console.log('null' || 'hello') // 'null'
```

Reply:
oh, unless `Global::Json` is setting the value explicit to `null` and not `'null'`?

Reply:
Correct, this is a hack to get the value `null`, _not_ the string `"null"`.